catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -397,8 +399,19 @@ private void setProxiesForImagery(List<Map<String, Object>> newImageryProviders)
         proxiedImageryProviders.clear();
         for (Map<String, Object> newImageryProvider : newImageryProviders) {
             HashMap<String, Object> map = new HashMap<>(newImageryProvider);
-            map.put(URL, SERVLET_PATH + "/" + urlToProxyMap.get(newImageryProvider.get(URL)
-                    .toString()));
+            String baseUrl = newImageryProvider.get(URL).toString();
+            boolean proxyEnabled = true;
+            Object proxyEnabledProp = newImageryProvider.get(PROXY_ENABLED);
+            if (proxyEnabledProp instanceof String) {
```

Comment:
Actually reverted this back to how it was. This is slightly different in that it will set it to false if the property is not there and this needs to default to `true`.